src/components/CsvUpload/CopyTemplateHeadersButtonGroup.tsx

```diff
@@ -0,0 +1,28 @@
+import { CopyIcon } from 'lucide-react'
+import { HStack } from '../ui/Stack/Stack'
+import { Button, ButtonVariant } from '../Button/Button'
+
+const copyTextToClipboard = (text: string) => {
+  navigator.clipboard.writeText(text).catch(() => {})
+}
+
+interface CopyTemplateHeadersButtonGroupProps {
+  templateHeaders: string[]
+}
+
+export const CopyTemplateHeadersButtonGroup = ({ templateHeaders }: CopyTemplateHeadersButtonGroupProps) => {
+  return (
+    <HStack gap='3xs' className='Layer__csv-upload__copy-template-headers-button-group'>
+      {templateHeaders.map(header => (
+        <Button
+          key={header}
+          onClick={() => copyTextToClipboard(header)}
```

Comment:
Nice touch

Reply:
I recognized the 'copy' icon but I expect most of users _won't_. Think it's worth doing a tooltip when hovered or after clicked?